packit_service/worker/github_handlers.py

```diff
@@ -247,10 +247,14 @@ def handle_pull_request(self):
         default_project_name = (
             f"{self.project.namespace}-{self.project.repo}-{self.event.pr_id}"
         )
+        collaborators = self.project.who_can_merge_pr()
         project = self.job.metadata.get("project") or default_project_name
         owner = self.job.metadata.get("owner") or self.api.copr.config.get("username")
         r = BuildStatusReporter(self.project, self.event.commit_sha)
-
+        if self.event.github_login not in collaborators:
```

Comment:
do we want to make this configurable? so that upstream repos would say that they want to check PRs submitted by anyone?

Reply:
Personally, I don't want to allow it. Like in `sclorg` case. Some have to call trigger.

Reply:
fair enough, but I'm pretty sure we'll receive such request fairly soon